app/src/main/kotlin/org/akanework/gramophone/logic/GramophoneExtensions.kt

```diff
@@ -98,6 +100,21 @@ fun MediaItem.getFile(): File? {
     return getUri()?.toFile()
 }
 
+fun MediaItem.getBitrate(): Long? {
+    val retriever = MediaMetadataRetriever()
+    return try {
+        val filePath = getFile()?.path ?: return null
+        retriever.setDataSource(filePath)
+        retriever.extractMetadata(MediaMetadataRetriever.METADATA_KEY_BITRATE)
+            ?.toLongOrNull()
+    } catch (e: Exception) {
+        Log.w("getBitrate", e.printStackTrace().toString())
```

Comment:
```suggestion
        Log.w("getBitrate", Log.getStackTraceString(e))
```
Sorry for the typo earlier lol, after this can merge

Reply:
No problem